apps/condo/domains/miniapp/schema/AllMiniAppsService.test.js

```diff
@@ -2,225 +2,382 @@
+                    await allMiniAppsByTestClient(anonymous, organization)
                 })
-                const [data] = await allMiniAppsByTestClient(client, client.organization.id)
-                expect(data).toBeDefined()
-                expect(data).not.toEqual(expect.arrayContaining([{
-                    id: globalIntegration.id,
-                    type: B2B_APP_TYPE,
-                }]))
             })
         })
-        describe('Display priority and labeling', () => {
-            test('All miniapps must be ordered by displayPriority', async () => {
-                const admin = await makeLoggedInAdminClient()
-                const [firstApp] = await createTestB2BApp(admin, { isHidden: false, displayPriority: 6 })
-                const [secondApp] = await createTestB2BApp(admin, { isHidden: false, displayPriority: 3 })
-                const [thirdApp] = await createTestB2BApp(admin, { isHidden: false, displayPriority: 4 })
-                const [forthApp] = await createTestB2BApp(admin, { isHidden: false, displayPriority: 3, label: APP_NEW_LABEL })
-
-                const client = await makeEmployeeUserClientWithAbilities()
-                const [data] = await allMiniAppsByTestClient(admin, client.organization.id)
-                expect(data).toBeDefined()
-                expect(data).toEqual(expect.arrayContaining([
-                    expect.objectContaining({ id: firstApp.id, type: B2B_APP_TYPE, label: null }),
-                    expect.objectContaining({ id: secondApp.id, type: B2B_APP_TYPE, label: null }),
-                    expect.objectContaining({ id: thirdApp.id, type: B2B_APP_TYPE, label: null }),
-                    expect.objectContaining({ id: forthApp.id, type: B2B_APP_TYPE, label: APP_NEW_LABEL }),
-                ]))
+    })
+    describe('Data shown', () => {
+        test('Must include all necessary fields', async () => {
+            const [app, appAttrs] = await createTestB2BApp(support, {
+                isHidden: false,
+                category: SMART_HOME_CATEGORY,
+                icon: 'House',
+                label: APP_NEW_LABEL,
+                menuCategory: PROPERTIES_CATEGORY,
+                logo: new UploadingFile(path.resolve(conf.PROJECT_ROOT, 'apps/condo/domains/common/test-assets/dino.png')),
+            })
+            const [data] = await allMiniAppsByTestClient(employee, organization)
+            expect(data).toEqual(expect.arrayContaining([
+                {
+                    id: app.id,
+                    connected: false,
+                    accessible: false,
+                    name: appAttrs.name,
+                    shortDescription: appAttrs.shortDescription,
+                    category: appAttrs.category,
+                    icon: appAttrs.icon,
+                    label: appAttrs.label,
+                    menuCategory: appAttrs.menuCategory,
+                    logo: expect.stringContaining(''),
```

Comment:
I'm expecting to contain any string (any because filename can be changed depending on file adapter). There's no `expect.anyString` in jest, but there's `stringContaining`. So I've used a trick, that any string contains empty string :)